showcase/java/src/test/java/com/google/api/showcase/ShowcaseTest.java

```diff
@@ -258,4 +277,66 @@ public void block() {
                 .build());
     assertThat(result.getContent()).isEqualTo("Hello, World!");
   }
+
+  @Test
+  public void attemptSequence() {
+    Sequence toCreate =
+        Sequence.newBuilder()
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setStatus(Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE)))
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setStatus(Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE)))
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setStatus(Status.newBuilder().setCode(Code.OK_VALUE)))
+            .build();
+    Sequence sequence = seqClient.createSequence(toCreate);
+    assertThat(sequence.getName()).isNotNull();
+
+    seqClient.attemptSequence(sequence.getName());
+
+    SequenceReport report = seqClient.getSequenceReport(sequence.getName() + "/sequenceReport");
+    assertThat(report.getAttemptsList()).isNotNull();
+    assertThat(report.getAttemptsList().size()).isEqualTo(3);
+  }
+
+  @Test
+  public void attemptSequenceTimeoutBackoff() {
+    Sequence toCreate =
+        Sequence.newBuilder()
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setDelay(Duration.newBuilder().setSeconds(2L))
+                    .setStatus(Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE)))
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setDelay(Duration.newBuilder().setSeconds(2L))
+                    .setStatus(Status.newBuilder().setCode(Code.UNAVAILABLE_VALUE)))
+            .addResponses(
+                Sequence.Response.newBuilder()
+                    .setStatus(Status.newBuilder().setCode(Code.OK_VALUE)))
+            .build();
+    Sequence sequence = seqClient.createSequence(toCreate);
+    assertThat(sequence.getName()).isNotNull();
+
+    seqClient.attemptSequence(
+        AttemptSequenceRequest.newBuilder().setName(sequence.getName()).build());
+
+    SequenceReport report = seqClient.getSequenceReport(sequence.getName() + "/sequenceReport");
+    assertThat(report.getAttemptsList()).isNotNull();
+    assertThat(report.getAttemptsList().size()).isEqualTo(3);
+    for (int i = 0; i < report.getAttemptsList().size(); i++) {
+      if (i == 0) {
```

Comment:
Why not just `int i = 1` in the for loop declaration?

Reply:
Good call, changed.